client/src/components/RelatedProducts/RelatedProducts.jsx

```diff
@@ -20,79 +22,85 @@ const RelatedProducts = ({ productID }) => {
       })
     }, [])
 
-    console.log('this is related producst', relatedProducts);
-
-    useEffect(() => {
-      if(relatedProducts.length > 0) {
-        getProductDataAsync();
-        getProductStyleAsync();
-      }
-    }, [relatedProducts])
-
-    const getProductDataAsync = async () =>{
-      const tempData =[];
-      for (var relatedProductId of relatedProducts) {
-       const data = await getProductData(relatedProductId);
-       tempData.push(data);
-      }
-      setProductData(tempData);
+  useEffect(() => {
+    if(relatedProducts.length > 0) {
+      getProductDataAsync();
+      getProductStyleAsync();
     }
+  }, [relatedProducts])
 
-    const getProductData = (relatedProduct) => {
-     return axios.get(`https://app-hrsei-api.herokuapp.com/api/fec2/hr-sfo/products/${relatedProduct}`, {headers: config})
-        .then((product) => {
-          return product.data;
-        })
-        .catch(() =>{
-          console.log(response.err, 'err at req2')
-        })
+  const getProductDataAsync = async () =>{
+    const tempData =[];
+    for (var relatedProductId of relatedProducts) {
+      const data = await getProductData(relatedProductId);
+      tempData.push(data);
     }
+    setProductData(tempData);
+  }
 
-    console.log('this is product data', productData);
-
-    const getProductStyleAsync = async () => {
-      var tempData = [];
-      for (var relatedProductStylesId of relatedProducts) {
-        const data = await getProductStyle(relatedProductStylesId);
-        tempData.push(data);
-      }
-      console.log(tempData, 'this is tempdata')
-      setProductStyle(tempData);
-    }
+  const getProductData = (relatedProduct) => {
+    return axios.get(`https://app-hrsei-api.herokuapp.com/api/fec2/hr-sfo/products/${relatedProduct}`, {headers: config})
```

Comment:
can at some point refactor the url and header information out into a variable you refer to instead of typing the whole thing out multiple times for axios requests.

Reply:
ill make sure to get on this in a later branch